app/views/requests/requests.php

```diff
@@ -0,0 +1,22 @@
+<h2>Make an asset request</h1>
+<?php
+if ($requested !== null) {
+    if ($requested == true) {
+        echo "Requested successfully";
+    }
+    else {
+        echo "Submission failed";
+    }
+    echo "<br><br>";
+}
+?>
+<form action="" method="POST">
+    <fieldset>
+        <legend>Common details about the person</legend>
+        <label for="organisation_id">Organisation Id: </label><input id="organisartion_id" name="details[organisation_id]" required /><br>
+        <label for="asset_id">Asset: </label><input id="asset_id" name="details[asset_id]" required /><br>
```

Comment:
It doesn't make sense if someone directly have to enter `organisartion_id` and `asset_id`.

We need to think of a better way for this.